.profile

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+### Put your custom scripts to run in bash login
+
+localectl set-keymap us
```

Comment:
I think we can directly add this to app.yml similar to https://github.com/dincho/pagrant/blob/master/ansible/app.yml#L14